sdk/tests/conformance/extensions/oes-texture-half-float.html

```diff
@@ -173,17 +174,49 @@
             runRenderTest(format, f.subtractor, uint16Data);
         });
 
-        // Check if attaching texture as FBO target succeeds (Not mandatory)
-        runRenderTest(gl.RGBA, [10000, 10000, 10000, 10000], null);
-        runRenderTest(gl.RGB, [10000, 10000, 10000, 1], null);
+        (function() {
+            debug("");
+            var renderableExtName = "EXT_color_buffer_half_float";
+            if (isRenderable(gl, ext)) {
+                var supported = gl.getSupportedExtensions();
+                assertMsg(supported.includes(renderableExtName),
+                          "Implicit renderability requires exposing support for " + renderableExtName);
+            } else {
+                var renderableExt = gl.getExtension(renderableExtName);
```

Comment:
Should we just assert the extension isn't included in the supported extensions here?